Update discussion based on cover letter, and check with grammarly the entire document (#38)
greenelab/phenoplier_manuscript: content/05.discussion.md

Our key innovation is that we project gene-trait associations through a latent representation derived not strictly from measures of normal tissue but also cell types under a variety of stimuli and at various developmental stages.
This improves interpretation by going beyond statistical associations to infer cell type-specific features of complex phenotypes.
We found that our approach can identify disease-relevant cell types from summary statistics, and several disease-associated gene modules were replicated in eMERGE.
Using a CRISPR screen to analyze lipid regulation, we found that our gene module-based approach can prioritize causal genes even when single gene associations are not detected.
We interpret these findings with an omnigenic perspective of "core" and "peripheral" genes, suggesting that the approach can identify genes that directly affect the trait with no mediated regulation of other genes, and thus prioritize alternative and potentially more attractive therapeutic targets.
Using our gene module perspective, we also integrated drug-induced transcriptional profiles, which allowed us to connect diseases, drugs, and cell types.

> We found that our method predicted FDA-approved drugs for various complex diseases more accurately than a single-gene approach.

Reviewer: I am not sure how much it makes sense to hang your hat on this. I think it's less important that it's better (and not an argument we really want to have). It's just interesting that you can throw out a bunch of signal and it's not worse. Maybe not key to emphasize in the intro/discussion and instead something to deal with specifically in the results in a careful way? I could see this as something that a reviewer could really go after.

Author: Yes, I agree. I removed the last sentence.

Furthermore, and beyond statistical prediction, we focused on a particular drug (niacin) and set of traits (cardiovascular diseases) to show that the approach connects disease-relevant transcriptional processes with known mechanisms of action.

> This suggests that our proposal could also be helpful in better understanding the mechanisms of pharmacological effect of known or experimental drugs.

Reviewer: ```suggestion
This suggests that the conceptual approach may reveal the mechanisms of pharmacological effect of known or experimental drugs.
```
